api/data/locking.go

```diff
@@ -48,3 +51,62 @@ type (
 		ByPassedGovernance bool
 	}
 )
+
+// Encode marshal ObjectLockConfiguration to string.
+func (conf *ObjectLockConfiguration) Encode() string {
```

Comment:
Can be https://pkg.go.dev/encoding#TextMarshaler then. It's not very popular, but it exists.